apis/python/src/tiledbsoma/raw_group.py

```diff
@@ -82,7 +82,7 @@ def from_anndata(
         self._add_object(self.var)
```

Comment:
Do `raw`'s `var`/`varm` slots need to be decategoricalized?

Reply:
good call, thanks @aaronwolen ! :)